openmc/universe.py

```diff
@@ -53,6 +53,10 @@ def __repr__(self):
     def name(self):
         return self._name
 
+    @property
```

Comment:
True, there is no setter, but this property helps to ensure that the `Universe`'s contained cells are modified with the `add_cell`/`remove_cell` methods rather than being modified directly.

Reply:
Actually, I suppose those methods can be worked around by modifying the object returned by the property, but it does avoid setting it to something like

```python
universe.cells = [openmc.Cell(), openmc.Cell()]
```